src/lib.rs

```diff
@@ -14,7 +14,8 @@
 //!
 //! ## Rust Version
 //!
-//! This version of indexmap requires Rust 1.18 or later.
+//! This version of indexmap requires Rust 1.18 or later, or 1.30+ for
+//! development builds.
```

Comment:
Nice catch. It's unfortunately reasonable to say it either way - the minimum Rust version is 1.18 if you trust it continues to work and 1.30 if you a crate that actually has tests.

I don't know how silly the discussion is, but I'm a bit uncomfortable with saying that 1.18 is a supported version. But it should be ok, IMO it's not worth dwelling on, at this point.

Reply:
Fair enough. The travis at least tests what we claim here (builds on 1.18, tests on 1.30).